### PR TITLE
Adding documentation for Web Api in Orchard

### PR DESCRIPTION
**Pull request by wharri220**
Done on behalf of Radio Systems Corporation

### CODE REVIEW

**Comment by wharri220** [comment]
Indeed a typo, fixed that. Thanks!

File: Documentation/WebApi-In-Orchard.markdown [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+		}
+	}
+	
+Now, the Api endpoint can be reached by hitting "/api/myapi".
+
+## Configuring Web Api in Orchard
+
+Since Orchard doesn't have the concept of an AppStart file, in order to add custom configuration to Web Api in Orchard, you must do so in an Autofac module. For example, the following will set the default Web Api return type to Json, and will ensure that Json objects/properties returned by the Api follow the camelCased naming convention.
+
+	namespace Blue.Core {
```
Comment (agriffard) [inline Documentation/WebApi-In-Orchard.markdown]: You should have kept the same namespace than your example : MyCustomModule

**Comment by agriffard** [comment]
It reminds me this article : http://sebastienros.com/web-api-makes-it-in-orchard

**Comment by wharri220** [comment]
That is a good article about Web API in Orchard from one of the Orchard project's core developers. Very useful information. I've seen very similar information on several blog posts and articles, and as a developer actively working on Orchard projects, I've gained much knowledge of using Web Api within Orchard for myself. I felt the information needed to be included in the documentation for the Orchard project. It will be more useful to new Orchard developers who aren't as knowledgeable of the community. And it's often been stated in the Orchard community that any additions to the documentation is greatly needed and appreciated, as it's somewhat lacking.

**Pull request merged**